en/index.md

```diff
@@ -524,6 +524,46 @@ For example:
 
 - **Don’t use:** Click the ![pencil](https://assets.ubuntu.com/v1/633aab2b-pencil.png) pencil icon to edit the post.
 
+## Hyperlinks
+
+Here are some pointers about the general usage of hyperlinks and how to format them correctly.
+
+### General usage
+Avoid excessive links in the same paragraph or instruction. If you find yourself introducing several links in your content, consider listing them in a separate section called "Related topics", "Additional resources", or similar.
+
+When possible, avoid external links. They can easily become outdated, making the document harder to maintain.
+
+When linking to versioned files or specific lines of code, copy the **permalink** instead of the URL. This will ensure the link is bound to the current revision of the file, so it will direct to the same content even if the file changes.
```

Comment:
This is something that's really hard to implement in Discourse. On GH/RTD we can do this easily with anchors and with intersphinx, but not on Discourse. So we might want to qualify this a bit - even if it's just to say "if there's a permalink available, use that".

Reply:
@akcano I'll keep this initial version very simple, but I'm adding that to my growing list of extra guidelines to discuss for a future revision of this section.